docx/oxml/text/paragraph.py

```diff
@@ -121,10 +138,12 @@ def style(self, style):
         pPr = self.get_or_add_pPr()
         pPr.style = style
 
-    def iter_r_lst_recursive(self):
+    def iter_r_lst_recursive(self, return_hyperlinks=False):
```

Comment:
This function is used in the upper level paragraph function to return all the runs and all the elements in the paragraph. In the case of returning all the runs, `return_hyperlinks` is false for backword compatibility. The second function that returns all elements in a paragraph is needed for the docx parser on platform.

I don't believe we should create two separate functions instead of this one because the majority of the code is the same, so I opted for this approach with flags.